Makefile

```diff
@@ -137,7 +129,7 @@ develop:
 	done
 
 link: develop
-	for PLUGIN in $(AVOCADO_EXTERNAL_PLUGINS); do\
+	for PLUGIN in $(AVOCADO_OPTIONAL_PLUGINS); do\
```

Comment:
This block is already inside `develop`, so there is no need to add it here. In fact, with this change, the `link` is not necessary anymore.